core/src/main/java/org/stellar/anchor/sep31/Sep31Service.java

```diff
@@ -151,6 +165,8 @@ public Sep31PostTransactionResponse postTransaction(
     updateDepositInfo(txn);
     sep31TransactionStore.save(txn);
 
+    StellarId senderStellarId = sep12CustomerStore.findById(txn.getSenderId()).toStellarId();
```

Comment:
This can be done by calling [`CustomerIntegration.getCustomer()`](https://github.com/stellar/java-stellar-anchor-sdk/blob/main/api-schema/src/main/java/org/stellar/anchor/api/callback/CustomerIntegration.java#L17)

Reply:
🤔 Can we consider not to store the customer information in the platform database to avoid multiple sources of truth?

Reply:
After updating the description of the fields in [Events Schema.yml](https://github.com/stellar/java-stellar-anchor-sdk/blob/main/docs/03%20-%20Implementing%20the%20Anchor%20Server/Communication/Events%20Schema.yml), seems like the database call wasn't needed.